### DISCUSSION TITLE
nuclei threw the error in template validation

### DISCUSSION THREAD

**Discussion by Jaypanchal1401** [projectdiscovery/nuclei#2525] [Q&A] [answered]
### Steps To Reproduce:
run nuclei with costume templates and its threw error like below

"Error occurred loading template /home/kali/Downloads/Nuclei-Templates-main/Find-wp-config-backup.yaml: invalid field format for 'id' (allowed format is ^([a-zA-Z0-9]+[-_])*[a-zA-Z0-9]+$)
[FTL] Could not validate templates: errors occured during template validation"

**Comment by jimen0** [comment]
Hi @Jaypanchal1401 - Please, file the issue template. Nuclei version is always important. Also, could you please share the contents of `/home/kali/Downloads/Nuclei-Templates-main/Find-wp-config-backup.yaml`? Without these, we can't know if it's your template that isn't well formatted or a nuclei issue.

**Comment by ehsandeep** [comment, marked as answer]
@Jaypanchal1401 this is expected error as the template in use does not have valid format for the `id` field, make sure to update the ID to not include space in it.

**Reply by Jaypanchal1401** [reply]
yes sir i checked it . there is no space in Id field . what should i do

**Reply by ehsandeep** [reply]
@Jaypanchal1401 share the template file so we can suggest / review!

**Reply by Jaypanchal1401** [reply]
https://github.com/clarkvoss/Nuclei-Templates
 this file contain all templates 
these all templates threw same error

**Reply by ehsandeep** [reply]
There is space in the ID field:

Current:

```yaml
id: wp-config backup Exposed
```

Expected:

```yaml
id: wp-config-backup-exposed
```

https://github.com/clarkvoss/Nuclei-Templates/blob/main/Find-wp-config-backup.yaml
